objects-api/src/main/kotlin/com/ritense/objectsapi/taak/TaakObjectConnector.kt

```diff
@@ -76,7 +92,7 @@ class TaakObjectConnector(
         val taakObject = TaakObjectDto(
             bsn = bsnProvider?.getBurgerServiceNummer(task),
             kvk = kvkProvider?.getKvkNummer(task),
-            verwerkerTaakId = UUID.fromString(task.executionId),
+            verwerkerTaakId = UUID.fromString(task.id),
```

Comment:
task.id isn't a UUID. Execution ID is. But can't there me multiple tasks in the execution?